.travis.yml

```diff
@@ -53,58 +47,95 @@ notifications:
-    server: https://test.pypi.org/legacy/
-    distributions: sdist bdist_wheel
-    user: plus3it
-    password:
-      secure: o1r3jJSHXoh+QjgYi0FCP3GAOJtEsgGdnW/oKYLCPfW9CQoSHXTZtH5MAYmyOT5c9TQLqKxMtQHu8rRx5ZQoAe5WOmdhzc85c01qMcbC0DW1LaSibGM3oqqDtPFEipoFry3CpJLVIURE2lRV56DJtfTv43DIrUpgwrGU1+ijtCWZdcEzQUizvNKpX+K5jGHty9zyUHnaMx/OiM16loVDeue1Ukh++aTEEW0dqk2eAhN8ihZc0uAhls4DcQgN6R+bLXoR7oBk84QzQtxJlNS6CQcVltmtWA2ARxVV6CUpEr/JegufdNI7/Vdg4THg0g/iu0TatfJy1m2MDopUWSGrUOz8BQRCw1reRwWfPjhvuU0Ny+EINl3cnER2CJ67njo9PRjH0bDFcJQe6umPU7lf3dUOvbXGWeEAYaF1BOn490K9XNpOqw4XcGtjnm3c4YBTSuCp95APji7G8HlWtN54K1UWFZN48IkvKW3WbZX95BgLB3131yITwBdnNaPHm7sgI1OS7V3+/T8eNr5EDXTvunVt0olPuWzcWuLVmRq6/qVwdrko9MHOsmgMSFJi/QNGF23XjfT+9Aa0MP8iGpEZF5b8ZeB0qHdeugw0uMDSZ87msR9wCbgcTCbub89/ZmNCbyqIUTWyarCTtXJ9ARrVqwjrAsTSP78wfizthAqHUt4=
-    skip_cleanup: true
-    skip_upload_docs: true
-    on:
-      branch: develop
-      repo: plus3it/watchmaker
-      condition: '"$TOXENV" == *"py27"*'
-  - provider: pypi
-    distributions: sdist bdist_wheel
-    user: plus3it
-    password:
-      secure: QBRj35p/WVBbfXWsOGaQfZ5LxLo2h/44iJRKaGkqAc/l7fNPubUcvKTvsuvFKDS68wdacnvSPifhVjJB8B/UwDyk3WqWmm/e7MBpWAJhZUina9qceeB7IPO53zSnfD8xAJMBe7iWRnMAQaKpPerSOKPpGn5qkdvfQXWhYHLe1X70RWVSkJlr35kkpBDvcqkKvKCi9EGrkJMbS+px/97n+b86wGKO0T8s2+YKAUh3EN9IHo6r9pG36uME2SCXqfhu9MH9aSlBLirOKvbvH+eVpOtAF+AfSsstMYFD2MbWEDYnXDrav4dqielk9o+Z2jaWvHAwBhCPp3aKYEx+xKrVsgu+9wE5e9i0T1CUPZSpeYkc+qPIjTZLmUpR9S32M19QmyewwyWPXHLD5TlFeHgU+/n0Y+dWJwBUESK58THvAXSewsTOhePEs+zuQfKXUFHHYYM+uJ020X3lv/R5SziSl93eD7nWuulBHwzBF3FEPiQnfrv1Ns0E28zdrkDJ6IxYES0tfHOHdFQZlSwGe8JJgzoXm2jtClJB/wkn/TwwInX7CvlgrVNrNDcHTGBiUoncb/yy6aIIR2DsCjYxd1WMN12DpaY+4OonDik/Ot4DcLiE1CA7b+1TKsbAv3G2YuFBp4QUBellnAU/L8wBxhgmhLTCntAQmPYuWezztRwEyG8=
-    skip_upload_docs: true
-    on:
-      tags: true
-      repo: plus3it/watchmaker
-      condition: '"$TOXENV" == *"py27"*'
-  - provider: releases
-    name: $WAM_VERSION
-    tag_name: $WAM_VERSION
-    target_commitish: $TRAVIS_COMMIT
-    body: $WAM_BODY
-    draft: false
-    skip_cleanup: true
-    api_key:
-      secure: NZQua4PHfAugtez+Kkp+TZxxg8sgFPuTnEmJmQ/BBazUTWy5IGcMJFcQxI6y+YVmiiMshwV36B53sQaY9lLwZEaPARjPMNprBCRuGR436s5UrkmDSAl2uH7kbpEkbsIA2L4V++85v0q2PVSacgWW8abGs0mHydA6chHKewlvKiyI5PqlEhgABPx5LDIF2Wmrg/YfdPxtmMb9IHP0LN1sj5x59dxe727fbY6/kUpb4mrQ306EHBYSMi4E2mpP2zIlrt21KUuU8iFHNaRKUzSlq5Lva4JeoKlh3RT7G3YA4u+7dH5eJUxnHFFhFnLBIz1d2wDY1tVaEGX1CDAKW6lSCBBOarnLnDkvtLDD23NVUNXC81+7OLObaw63rtubfC3syZ1Kp517R4vgIZwRhBrvvJDr2IO5X/YtHVETBwkhDb8Ol8djXrwRbOeEgGWwkU66BbgX2HlcoggQqDhDWZWBosu7iz01pXvAFKiylNHbPULNV2gOM9HzY0kmxq3/zzwdBfLxJH8delSCK1uvQpZlmaBW6SA0/l7yuoxDN8KrXSRgwuE9A3XuMe6ZhbU5AZSqWA7vMdnwCg05xK2zo3Xohi7QE4AxY1jZW2ficBWWWFcdUTZpG+TRKC8LzQWBmZ61s9lNn+HfFC5yqYkJopLPeRYzfyeUJN79rr9leC04sVI=
-    on:
-      branch: master
-      repo: plus3it/watchmaker
-      condition: '"$TOXENV" == *"py27"*'
+jobs:
+  include:
+    - stage: check
+      python: 3.6
+      env:
+        - TOXENV=check
+      install:
```

Comment:
i suspect we could remove the `install`, `script`, and `after_failure` sections, since they are the same as the settings at the top level, so they should get "inherited" when the stage section is merged into the job...